.github/actions/audit-accessibility/main.js

```diff
@@ -78,16 +78,6 @@ const audit = async (browser, url, disabledRules = []) => {
         .disableRules([
             // ignored because some stories don't include an H1 header
             'page-has-heading-one',
-            // ignored because we use invented autocomplete values to workaround related chrome issues
-            'autocomplete-valid',
-            // ignored because disabled input fields have a low contrast by design spec
-            'color-contrast',
```

Comment:
re-enabled and only disabled in some stories/skins (see disabledRulesByStoryAndSkin map)